cloudbuild.yaml

```diff
@@ -1,6 +1,10 @@
 steps:
 - name: gcr.io/cloud-builders/git
-  args: ['clone', 'https://github.com/polleyg/gcp-dataflow-copy-bigquery.git']
+  args: ['clone', '${_SOURCE_REPO}' ]
 
 - name: gcr.io/cloud-builders/gradle
-  args: ['build', 'run']
\ No newline at end of file
+  args: ['build', 'run' ,'-PappArgs="${_APP_ARGS}"']
```

Comment:
I thought we'd agreed dropping into the bash script that Gradle produces would be a nicer way to do this?